pkg/commands/process/settings/rules/ruby/lang/weak_encryption_with_data.yml

```diff
@@ -148,4 +148,8 @@ metadata:
     ## Resources
     - [BCrypt Explained](https://dev.to/sylviapap/bcrypt-explained-4k5c)
   dsr_id: DSR-7
+  cwe_id:
+    - 311
```

Comment:
I wouldn't say we're missing encryption. I think 326 covers the sensitive data part anyway so maybe drop this?